testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1296,23 +1296,6 @@
   end
 end
 
-When(/^I (enable|disable) the necessary repositories before installing Prometheus exporters on this "([^"]*)"((?: without error control)?)$/) do |action, host, error_control|
-  node = get_target(host)
-  os_version = node.os_version.gsub('-SP', '.')
-  os_family = node.os_family
-  # TODO: Check why tools_update_repo is not available on the openSUSE minion
-  repositories = os_family.match?(/^opensuse/) ? 'tools_pool_repo' : 'tools_pool_repo tools_update_repo'
-  if (os_family.match?(/^opensuse/) || os_family.match?(/^sles/)) && (product != 'Uyuni')
-    repositories.concat(' tools_additional_repo')
-    # Needed because in SLES15SP3 and openSUSE 15.3 and higher, firewalld will replace this package.
-    # But the tools_update_repo's priority doesn't allow to cope with the obsoletes option from firewalld.
-    if os_version.to_f >= 15.3
-      node.run('zypper addlock -r tools_additional_repo firewalld-prometheus-config')
```

Comment:
I'm not 100% sure if that still need. So, I need to run these changes in a pipeline, to assure this is fine.
But... I first need to merge and rebase my pending PRs: https://github.com/uyuni-project/uyuni/pull/9670 https://github.com/uyuni-project/uyuni/pull/9667